cowamm/kpi/tvl_by_chain_4096107.sql

```diff
@@ -0,0 +1,85 @@
+-- This query computes the total TVL evolution per day for all CoW AMM pools for a given chain
+-- Parameters
+--  {{blockchain}} - the target chain
+
+with pools as (
+    select
+        created_at,
+        token_1_address,
+        token_2_address,
+        address
+    from query_3959044
```

Comment:
Shouldn't we filter based on blcokchain here? There is a blockchain column in the table returned from 3959044 so that is straightforward to do